package.json

```diff
@@ -166,7 +164,7 @@
     "test": "cross-env NODE_ENV=test BABEL_ENV=test TZ=UTC jest --config ./jest/jest.config.js --maxWorkers 2 --passWithNoTests",
     "test:size": "bundlesize",
     "lint": "npm run lint:js && npm run lint:css && npm run lint:lockfile",
-    "lint:js": "npm run type-check && eslint . --ext .js,.ts,.tsx",
+    "lint:js": "npm run type-check && eslint . --ext .js,.ts,.tsx --fix",
```

Comment:
`--fix` I guess you forgot this

Reply:
no I didn't :-) Actually as I am having problems with the actual version of eslint and pretty...I am using this command quite often and I thought it would be helpful for another devs..especially before pushing

Reply:
That's actually the job of `husky` and `lint-staged`, for every filed changed `prettier` pass through each file.

```
 "lint-staged": {
    "relative": true,
    "linters": {
      "*.{js,tsx,ts}": [
        "eslint . --ext .js,.ts,.tsx",
        "prettier --write"
      ],
      "*": [
        "detect-secrets-launcher --baseline .secrets-baseline",
        "git add"
      ]
    },
    "ignore": [
      "*.json"
    ]
  },
````

The script `lint:js` is being executed in the CI, this might cause unexpected errors due the files are modified in such environment. We don't need the `fix` flag.

Reply:
ah yes..it's true! thank you!